src/installer/BeatPackageCompiler/BeatPackageCompiler.cs

```diff
@@ -150,40 +147,19 @@ static void Main(string[] args)
 
             var packageContents = new List<WixEntity>
             {
-                new DirFiles(Path.Combine(opts.PackageInDir, MagicStrings.Files.All), path =>
+                new Files(Path.Combine(opts.PackageInDir, MagicStrings.Files.All), path =>
```

Comment:
This is the line responsible for choosing all files including subfolders (recursive), instead of just the part top folder